js/tests/README.md

```diff
@@ -0,0 +1,18 @@
+## How does Bootstraps's test suite work?
+
+Bootstrap uses [QUnit](http://api.qunitjs.com/), a powerful, easy-to-use JavaScript unit test suite. Each plugin has a file dedicated to its tests in `js/unit/[plugin-name].js`.
+
+* `/test/unit/` contains the unit test files for each Bootstrap plugin.
```

Comment:
I'll trust your judgement on this. Revised to use relative paths from current location.